lib/checksrc.pl

```diff
@@ -725,6 +726,12 @@ sub scanfile {
                       "typedef'ed struct");
         }
 
+        if($nostr =~ /(.*)! +(\w|\()/) {
```

Comment:
Don't we want to use `\s+` to handle any type of whitespace?

Reply:
Oh, maybe? Can there actually be any other whitespace that we don't already warn about elsewhere?

Reply:
Good point, it would be superfluous.